strongbox-web-core/src/main/java/org/carlspring/strongbox/rest/LoggingManagementRestlet.java

```diff
@@ -1,21 +1,23 @@
 package org.carlspring.strongbox.rest;
 
-import io.swagger.annotations.*;
 import org.carlspring.logging.rest.AbstractLoggingManagementRestlet;
-import org.springframework.stereotype.Component;
 
 import javax.ws.rs.Path;
-import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import java.io.IOException;
 import java.io.InputStream;
 
+import io.swagger.annotations.*;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Component;
+
 /**
  * @author carlspring
  */
 @Component
 @Path("/logging")
 @Api(value = "/logging")
+@PreAuthorize("permitAll")
```

Comment:
@AlexOreshkevich ,
I  think this needs to be furter fine-grained. Not everyone will be allowed to configure or view logs.